jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/sys/DatabaseOps.java

```diff
@@ -195,6 +207,19 @@ public static void compact(DatasetGraphSwitchable container) {
             LOG.debug(String.format("Compact %s -> %s\n", db1.getFileName(), db2.getFileName()));
 
             compact(container, loc1, loc2);
+
+            if ( !Sys.isWindows && shouldDeleteOld ) {
+                Path loc1Path = IOX.asPath(loc1);
+                LOG.info("Deleting old database after successful compaction (old db path='" + loc1Path + "')...");
```

Comment:
Make this `LOG.debug` because the other message level debug.

Reply:
updates